1-foundations/1-types-and-syntax/foundations-s1.ipynb

```diff
@@ -1136,7 +1516,24 @@
    "outputs": [],
    "source": [
     "# Modify item with index 1 \n",
-    "pets[1] = \"wolf\"\n",
+    "pets[1] = 'wolf'\n",
+    "print('Variable pets:', pets, type(pets))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Erase item in list by index. Item returned\n",
+    "pet_pop = pets.pop(3)\n",
+    "# Erase item in list by value. None returned\n",
+    "pet_remove = pets.remove(\"cat\")\n",
```

Comment:
maybe we should add a note clarifying that if the list has more than one item with the value in the parentheses, `.remove()` will remove the first, or at least keep that in mind in case someone asks

Reply:
added comment re this in df53e47b167c4789141146096b75fa711bda9548